scripts/package-manager/src/is-ci.js

```diff
@@ -0,0 +1,9 @@
+function isCI() {
```

Comment:
we cannot use any 3rd party dependencies for preinstall hooks so keeping it simple and focused for our use-case